cmd/disaster-recovery/disaster_recovery_test.py

```
@@ -0,0 +1,335 @@
+    def _get_request_id(self, response):
+        """Extract request ID from response headers"""
+        for header in ['x-rh-insights-request-id', 'x-rh-request-id', 'x-request-id']:
+            if header in response.headers:
+                return response.headers[header]
+        return None
+
+    def _fetch_available_quickstart(self):
+        """Fetch an available quickstart from the API"""
+        url = f"{self.base_url}/api/quickstarts/v1/quickstarts?account={self.account_id}"
+
+        print(f"[INIT] Fetching available quickstarts from API...")
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)